arbitrum-docs/arbos/l1-to-l2-messaging.mdx

```diff
@@ -1,3 +1,32 @@
+import { BigNumber } from "@ethersproject/bignumber";
+import { useEffect, useState } from "react";
+
+export const AddressAliasHelper = () => {
+  const CONSTANT = BigNumber.from("0x1111000000000000000000000000000000001111");
```

Comment:
Let's instead use the `Address.applyAlias` method from the arbitrum-sdk (that will among other things handle the appropriate address validation; e.g., as is, the page errors on a non-hex alphanumerical input)

Reply:
oh haha didn't know there's this method